_data/AntoniKwok/AntoniKwok.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Antoni Wijaya
+institution: Bina Nusantara University
+profile_pic: antoni.jpg
```

Comment:
Git is case-sensitive. Could you please fix the file name? You could do one of the following:
1. Change `antoni.jpg` to `Antoni.jpg ` in the markdown file.
2. Rename `Antoni.jpg ` to `antoni.jpg`.